src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
@AnshulMalik This actually converts the user's rawMessage to a message model which is defined in actions.js It is required.